admission-webhook/integration_tests/integration_test.go

```diff
@@ -463,6 +463,45 @@ func TestPossibleToUpdatePodWithNewCert(t *testing.T) {
 	assert.Equal(t, expectedCredSpec0, extractContainerCredSpecContents(t, pod3, testName3))
 }
 
+func TestHappyPathWithHostnameRandomization(t *testing.T) {
+	testName := "happy-path-with-hostname-randomization"
+	credSpecTemplates := []string{"credspec-0"}
+	templates := []string{"credspecs-users-rbac-role", "service-account", "sa-rbac-binding", "simple-with-gmsa"}
+
+	testConfig, tearDownFunc := integrationTestSetup(t, testName, credSpecTemplates, templates)
```

Comment:
maybe I am missing something but where does the feature get turned on?

Reply:
looking at the test results. these tests are passing for the chart (where it was turned on by default but not the non-chart scenarios which makes sense.  See https://github.com/kubernetes-sigs/windows-gmsa/pull/155#discussion_r1805024413  for a way to turn this on for a subset of tests.  Probably can rename `integration-rotation-enabled` test to `integration-optional-features` and make it just one additional tests for now

Reply:
Good to know we can flip the flag on/off for test, will update